src/bundle/Resources/views/themes/admin/content/components/meta_fields.html.twig

```diff
@@ -5,7 +5,7 @@
 {% block sections %}
     {% for identifier in meta_fields %}
         {% embed '@ibexadesign/ui/component/anchor_navigation/section.html.twig' with {'form': form, 'identifier': identifier} %}
-            {% set data_id = 'ibexa-edit-content-sections-meta-' ~ identifier %}
+            {% set data_id = '#ibexa-edit-content-sections-meta-' ~ identifier %}
```

Comment:
Fixing usage adding `#` in relevant places should be preferred. This HTML DOM `id` attribute can be used outside of JS selector context, and that would not be possible without striping the prefix.

Additionally, `document.getElementById()` does not expect you to place `#` as prefix, so in such case it could be an error.

Reply:
@Steveb-p hmm, it does look like `#` is expected - https://github.com/ibexa/admin-ui/blob/main/src/bundle/Resources/views/themes/admin/ui/menu/anchor_menu.html.twig#L95.

Reply:
@barw4 on the contrary, file you linked shows in multiple places that `#` is added before it is used. In the particular place linked it overwrites the previous value.

Overall, I will agree to keep it as is, only because it is already a mess - and it is prefixed automagically in some places, while being manually added in others. And fixing that is outside of scope of what you're doing.